### ISSUE TITLE
[results]distribute pyscenic grn jobs on a lsf cluster farm

### ISSUE THREAD

**Issue by wangmeijiao** [aertslab/pySCENIC#237]
Hi,  it is quite convenient and efficient to perform gene regulation network estimate by pyscenic for cell population number less than 10 thousands.  But in my case (~12000 cells x ~24000 genes with 20 cpus on a local linux machine), it spend more than one week!!  my possible solution:  grnBoost2 with the newest python dask + distribution (2020.12.0) + dask_jobqueue (v0.7.1).  Indeed I got the TF-target-importance table after 2hours with 45 distributed cpu cores. my question:  I am not sure whether or not the TF-target-importance table produced in this way will be valid to next pyscenic steps (like ctx, aucell), and grnBoost2 seems only support the dask v1.0.0 on local machine and not suitable to computer grid environment??.
In conclusion, I need some elegant way to do pySCENIC on a LSF cluster.

########my conda environment :############

#pyscenic==0.10.3,dask==2020.12.0,distributed==2020.12.0,pandas==0.25.3,numpy==1.19.1,loompy==3.0.6,scanpy==1.6.0,anndata==0.7.4

**Comment by cflerin** [comment]
Hi @wangmeijiao , glad to hear that you got this working across multiple compute nodes, and that sounds like a great performance increase. It should be no problem at all to use that TF-target-importance table in the subsequent steps.

For reference the format is:
```
TF      target  importance
FOS     FOSB    150.32743963584093
FOS     ZFP36   130.4814596042488
FOS     VAPA    127.23979658784661
FOS     SOS2    116.65148441651921
FOS     RBMS1   116.60752659126497
FOS     LENG8   116.09648088705285
FOS     BEX4    97.69948904981003
FOS     SBDS    86.55066009926372
FOS     ATP5G2  85.6722130956471
```
In this case the file is tab-delimited (but could also be csv) and includes the header row.
